Documentation/CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+Building IoT on Windows
```

Comment:
Would it make sense to put windows specific instructions to something like:
Contributing-Windows.md and then point there from here?

I think this should contain OS-independent instructions

Reply:
you are right, in fact most of these instructions are os-independent. I will fix this in the next commit.